web/src/lib/components/shared-components/context-menu/context-menu.svelte

```diff
@@ -65,9 +65,7 @@
     aria-label={ariaLabel}
     aria-labelledby={ariaLabelledBy}
     bind:this={menuElement}
-    class:max-h-[100vh]={isVisible}
-    class:max-h-0={!isVisible}
-    class="flex flex-col transition-all duration-[250ms] ease-in-out outline-none"
+    class="{isVisible ? 'max-h-screen max-h-svh' : 'max-h-0'} flex flex-col transition-all duration-[250ms] ease-in-out outline-none overflow-auto"
```

Comment:
I put both 100vh and 100svh for retro compatibility.
I preferred svh instead of dvh because it should be more stable reading the [CSS standard](https://www.w3.org/TR/2021/WD-css-values-4-20211016/#dynamic-viewport-percentage-units).